Source: ghostscript
Upstream: probably (jbig2dec's and ghostscript's upstream are identical/close)
Reason: libpng 1.4
--- jbig2dec/configure.ac
+++ jbig2dec/configure.ac
@@ -45,7 +45,7 @@
   fi
   dnl libpng requires pow() which may be in libm
   AC_SEARCH_LIBS([pow], [m])
-  AC_CHECK_LIB([png], [png_check_sig], [
+  AC_CHECK_LIB([png], [png_create_write_struct], [
     AC_CHECK_LIB([z], [deflate], [
       AC_DEFINE(HAVE_LIBPNG, 1, [Define if libpng is available (-lpng)])
       LIBS="-lpng -lz $LIBS"
